xml/System.Text/Encoding.xml

```diff
@@ -865,10 +865,18 @@
   
 ## Remarks  
   
-> [!WARNING]
->  Different computers can use different encodings as the default, and the default encoding can even change on a single computer. Therefore, data streamed from one computer to another or even retrieved at different times on the same computer might be translated incorrectly. In addition, the encoding returned by the <xref:System.Text.Encoding.Default%2A> property uses best-fit fallback to map unsupported characters to characters supported by the code page. For these two reasons, using the default encoding is generally not recommended. To ensure that encoded bytes are decoded properly, you should use a Unicode encoding, such as <xref:System.Text.UTF8Encoding> or <xref:System.Text.UnicodeEncoding>, with a preamble. Another option is to use a higher-level protocol to ensure that the same format is used for encoding and decoding.  
+## Remarks  
   
- The system ANSI code page defined by <xref:System.Text.Encoding.Default%2A> covers the ASCII set of characters, but the encoding is different from the encoding for ASCII. Because all <xref:System.Text.Encoding.Default%2A> encodings lose data, you might use <xref:System.Text.Encoding.UTF8%2A> instead. UTF-8 is often identical in the U+00 to U+7F range, but can encode other characters without loss.  
+> [!WARNING]
+>  Different computers can use different encodings as the default, and the default encoding can change on a single computer. If you use the <xref:System.Text.Encoding.Default%2A> encoding to encode and decode data streamed between computers or retrieved at different times on the same computer, it may translate that data incorrectly. In addition, the encoding returned by the <xref:System.Text.Encoding.Default%2A> property uses best-fit fallback to map unsupported characters to characters supported by the code page. For these reasons, using the default encoding is not recommended. To ensure that encoded bytes are decoded properly, you should use a Unicode encoding, such as <xref:System.Text.UTF8Encoding> or <xref:System.Text.UnicodeEncoding> with a preamble. You could alsouse a higher-level protocol to ensure that the same format is used for encoding and decoding.  
+
+### The Default property in the .NET Framework
+
+In the .NET Framework on the Windows desktop, the <xref:System.Text.Encoding.Default%2A> property always gets the system's active code page and creates a <xref:System.Text.Encoding> object that corresponds to it. This ANSI code page includes the ASCII character set along with additional characters, which vary by code page. Because all <xref:System.Text.Encoding.Default%2A> encodings lose data, consider using the <xref:System.Text.Encoding.UTF8%2A> encoding instead. UTF-8 is often identical in the U+00 to U+7F range, but can encode characters outside the ASCII range without loss.
```

Comment:
>This ANSI code page includes  [](start = 213, length = 29)

Windows 10 in the near future will start support UTF8 as active codepage. so I would like to avoid ANSI in here